sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/CoverageReportImportSensor.java

```diff
@@ -108,7 +108,6 @@ void analyze(SensorContext context, Coverage coverage) {
       } else if (inputFile.type().equals(Type.TEST)) {
         fileCountStatistics.test++;
         LOG.debug("Skipping '{}' as it is a test file.", filePath);
-        // Do not log for test files to avoid pointless noise
       } else if (!coverageConf.languageKey().equals(inputFile.language())) {
```

Comment:
This comment is out of date. The logging was added in #2874 but the old comment wasn't removed.